vendor/github.com/hashicorp/go-azure-sdk/resource-manager/applicationinsights/2022-04-01/applicationinsights/README.md

```diff
@@ -77,7 +77,7 @@ if model := read.Model; model != nil {
 
 ```go
 ctx := context.TODO()
-id := applicationinsights.NewResourceGroupID()
+id := applicationinsights.NewResourceGroupID("12345678-1234-9876-4563-123456789012", "example-resource-group")
```

Comment:
why are we editing a vendor'd file?

Reply:
This file is updated by `go-azure-sdk`,  after I resolve the file conflict with the latest code,  vendor files need to be updated.
https://github.com/hashicorp/go-azure-sdk/blob/main/resource-manager/applicationinsights/2022-04-01/applicationinsights/README.md?plain=1#L80

Reply:
merge the latest code, this diff has been reoslved.